valio-master/Valio/valio.json

```diff
@@ -0,0 +1,196 @@
+[
+  {
+    "title": "Thursday",
+    "subtitle": "June 5th",
```

Comment:
I would use a timestamp here or a date in some format, instead of title being weekday, subtitle being month and day.
Data should be independent of how it's presented. The app should generate the title & subtitle based on `NSDate` and `NSCalendar`. It would be easier to translate too.